suite-common/wallet-core/src/send/sendFormSolanaThunks.ts

```diff
@@ -123,6 +139,13 @@ const fetchAccountOwnerAndTokenInfoForAddress = async (
     return [accountOwner, tokenInfo] as const;
 };
 
+function assertIsSolanaAccount(
+    account: Account,
+): asserts account is Extract<Account, { networkType: 'solana' }> {
+    if (account.networkType !== 'solana')
+        throw new Error(`Invalid network type. ${account.networkType}`);
+}
+
```

Comment:
```suggestion
const assertIsSolanaAccount = (
    account: Account,
): asserts account is Extract<Account, { networkType: 'solana' }> => {
    if (account.networkType !== 'solana') {
        throw new Error(`Invalid network type. ${account.networkType}`);
    }
}

```

arrow function + brackets, then please squash

Reply:
I would, but functions that return `asserts` has to be function declarations
https://blog.logrocket.com/assertion-functions-typescript/#:~:text=Learn%20more%20%E2%86%92-,Function%20declarations%20and%20expressions,-At%20the%20time